client/src/Components/CommonComponent/Sidebar/index.jsx

```diff
@@ -0,0 +1,259 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Icon } from 'antd';
+
+import './style.css';
+
+export default class Sidebar extends Component {
+  state = {
+    accounts: 'hidden',
+    reports: 'hidden',
+    msg: 'hidden',
+    settings: 'hidden',
+    arrow_down_accounts:'inline',
+    arrow_up_accounts:'hidden',
+    arrow_down_reports:'inline',
+    arrow_up_reports:'hidden',
+    arrow_down_msg:'inline',
+    arrow_up_msg:'hidden',
+    arrow_down_settings:'inline',
+    arrow_up_settings:'hidden',
+  }
+  handleClick = (value) => (e) => {
+    this.setState((prev) => {
+      let display = '';
+      let up='';
+      let down='';
+      if (prev[value] === 'hidden') {
+        display = 'block';
+        up='inline';
+        down='hidden';
+      } else {
+        display = 'hidden';
+        up='hidden';
+        down='inline';
+      }
+      if(value==='accounts')
+      return { [value]: display , arrow_down_accounts:down,arrow_up_accounts:up};
+      if(value==='reports')
+      return { [value]: display , arrow_down_reports:down,arrow_up_reports:up};
+      if(value==='msg')
+      return { [value]: display , arrow_down_msg:down,arrow_up_msg:up};
+      if(value==='settings')
+      return { [value]: display , arrow_down_settings:down,arrow_up_settings:up};
+    })
+  }
+  render() {
+      
+    const { accounts, reports, msg, settings,arrow_down_accounts,arrow_up_accounts ,arrow_down_reports,arrow_up_reports,arrow_down_msg,arrow_up_msg,arrow_down_settings,arrow_up_settings} = this.state;
+    return (
+      <div className="container">
+
+        <div className="sidebar">
+          <h1 className="sidebar__header">زاجل</h1>
+          <div className="sidebar__bar">
+            <div className='home'>
+              <Link to="/" className="sidebar__links ">
+                <span>
+                  الرئيسية
+            <Icon type="home" style={{marginLeft:'7px'}}/>
```

Comment:
do not use inline style can put a class name for this component and use it in your stylesheet.
apply it for all inline style

Reply:
But if i used classname l will need deferent class for each one
And i try use a class the style didn't apply

Reply:
- no you use to style 7px and 5px for Icon 
- mmm but I apply it and the style apply